code/cmedecoder/tall_book.q

```diff
@@ -27,7 +27,7 @@ qtf:{[x;d]
   };
 
 .cme.tallbook:{[qt]
-  d:exec Symbol!MarketDepth from .raw.definitions;
+  d:exec Symbol!.raw.dfltlvl^MarketDepth from .raw.definitions;
```

Comment:
I don't think this change is needed (or won't do anything)? I think `.raw.dfltlvl` is only needed in the case were a symbol isn't in `.raw.definitions`, in which case this still won't do anything? Might be better to put this in next line in the `setbook` call

Reply:
Added a new commit that works for tallbook as well that is the result:
![tallbooknewfix](https://user-images.githubusercontent.com/51996872/61786315-52478880-ae05-11e9-966e-41f35a329266.jpg)